src/Accounts/Accounts/help/Get-AzAccessToken.md

```diff
@@ -8,12 +8,12 @@ schema: 2.0.0
 # Get-AzAccessToken
 
 ## SYNOPSIS
-Get secure raw access token. When using -ResourceUrl, please make sure the value does match current Azure environment. You may refer to the value of `(Get-AzContext).Environment`.
+Get secure access token. When using -ResourceUrl, please make sure the value does match current Azure environment. You may refer to the value of `(Get-AzContext).Environment`.
 
 > [!NOTE]
-> For security purposes, the default output type will change from a plain text `String` to
-> `SecureString`. To prepare for this change and ensure secure handling, use the **AsSecureString**
-> parameter before the update takes effect.
+> For security purposes, the default output type changes from a plain text `String` to `SecureString`.
+> Please refer to [Frequently asked questions about Azure PowerShell](https://learn.microsoft.com/en-us/powershell/azure/faq)
+> for how to convert from `SecureString` to plain text.
```

Comment:
```suggestion
> For security purposes, the default output type has been changed from a plain text `String` to `SecureString`.
> Please refer to [Frequently asked questions about Azure PowerShell](https://learn.microsoft.com/en-us/powershell/azure/faq)
> for how to convert from `SecureString` to plain text.
```

@mikefrobbins could you help review this now that we've updated `Get-AzAccessToken` to output `SecureString`? BTW do we have a fwlink for the FAQ page